src/models/subscription.ts

```diff
@@ -247,7 +247,7 @@ export default class Subscription extends Sequelize.Model {
 		this.changed("repoSyncState", true);
 		await this.save();
 
-		if (await booleanFlag(BooleanFlags.NEW_REPO_SYNC_STATE, true)) {
+		if (await booleanFlag(BooleanFlags.NEW_REPO_SYNC_STATE, true, this.jiraHost)) {
```

Comment:
AFAIK we should fallback to false, shouldn't we? 🌶️

Reply:
yeah, probably.  We really need to fix this for local dev though :/

Reply:
I use dev launch darkly for local devloop, there is an instruction on how to do it in confluence go/arc-ops -> howto feature flags